### PR TITLE
Added -doc subpackage. Lots of fixes. Fixed exclusion of plugins.

### PR DESCRIPTION
**Pull request by xvitaly**
* Added -doc subpackage.
* Fixed exclusion of plugins in main package.
* Corrected URLs in Source so it can be now downloaded by spectool.
* Removed legacy code from SPEC: Group, %dir, %defattr, etc.

### CODE REVIEW

File: pycharm-community.spec [hunk 1]
```diff
@@ -76,15 +67,21 @@ for productive Python development on all levels
 
 %package plugins
 Summary:       Plugins for intelligent Python IDE
-Group:         System Environment/Libraties
-Requires:      %{name}
-Requires:      %{name} = %{version}
+Requires:      %{name}%{?_isa} = %{version}-%{release}
+
+%package doc
```
Comment (ignatenkobrain) [inline pycharm-community.spec]: could be moved after `%description plugins`

File: pycharm-community.spec [hunk 2]
```diff
@@ -76,15 +67,21 @@ for productive Python development on all levels
...
+Summary:       Documentation for intelligent Python IDE
+BuildArch:     noarch
+Requires:      %{name} = %{version}-%{release}
```
Comment (ignatenkobrain) [inline pycharm-community.spec]: should not require base package

File: pycharm-community.spec [hunk 3]
```diff
@@ -141,72 +172,46 @@ desktop-file-install                          \
...
-%dir %{_javadir}/%{name}/%{plugins_dir}/svn4idea/*
-%{_javadir}/%{name}/%{plugins_dir}/svn4idea/*
-%dir %{_javadir}/%{name}/%{plugins_dir}/tasks/*
-%{_javadir}/%{name}/%{plugins_dir}/tasks/*
-%dir %{_javadir}/%{name}/%{plugins_dir}/terminal/*
-%{_javadir}/%{name}/%{plugins_dir}/terminal/*
-%exclude %{_javadir}/%{name}/%{plugins_dir}/{BashSupport,CppTools,idea-markdown}/*
-%exclude %{_javadir}/%{name}/%{plugins_dir}/{intellij-ansible,markdown,gitlab-integration-plugin}/*
-%exclude %{_javadir}/%{name}/%{plugins_dir}/{Go,IdeaVim,idea-multimarkdown,editorconfig,ini4idea}/*
+%{_javadir}/%{name}
```
Comment (phracek) [inline pycharm-community.spec]: I guess, %dir %{_javadir}/%{name} should be there. Package has to own the directory.

**Comment by phracek** [comment]
Did you try to run ./build.sh script? Does it work with your changes?

**Comment by xvitaly** [comment]
> Did you try to run ./build.sh script? Does it work with your changes?

This script is now using spectool. Now works fine for me:
```
Downloading sources...
Getting http://download.jetbrains.com/python/pycharm-community-2016.2.tar.gz to ./pycharm-community-2016.2.tar.gz
Getting https://plugins.jetbrains.com/files/4230/26121/BashSupport-1.5.8.162.zip#/BashSupport-1.5.8.162.zip to ./BashSupport-1.5.8.162.zip
Getting https://plugins.jetbrains.com/files/164/22030/ideavim-0.44-297.zip#/ideavim-0.44-297.zip to ./ideavim-0.44-297.zip
Getting https://plugins.jetbrains.com/files/7294/24766/editorconfig-145.258.3.zip#/editorconfig-145.258.3.zip to ./editorconfig-145.258.3.zip
Getting https://plugins.jetbrains.com/files/6981/27026/ini4idea-162.1121.34.zip#/ini4idea-162.1121.34.zip to ./ini4idea-162.1121.34.zip
Getting https://plugins.jetbrains.com/files/1373/19624/CppTools.zip#/CppTools-0.8.8.zip to ./CppTools-0.8.8.zip
Getting https://plugins.jetbrains.com/files/5047/25366/Go-0.11.1295.zip#/Go-0.11.1295.zip to ./Go-0.11.1295.zip
Getting https://github.com/nicoulaj/idea-markdown/archive/0.9.7.zip#/idea-markdown-0.9.7.zip to ./idea-markdown-0.9.7.zip
Getting https://plugins.jetbrains.com/files/7793/25156/markdown-2016.1.20160405.zip#/markdown-2016.1.20160405.zip to ./markdown-2016.1.20160405.zip
Getting https://plugins.jetbrains.com/files/7792/25063/intellij-ansible.zip#/intellij-ansible-0.9.3.zip to ./intellij-ansible-0.9.3.zip
Getting https://plugins.jetbrains.com/files/7447/17542/gitlab-integration-plugin.zip#/gitlab-integration-plugin-1.0.6.zip to ./gitlab-integration-plugin-1.0.6.zip
Getting https://plugins.jetbrains.com/files/7724/27184/Docker-plugin.zip#/Docker-plugin-2.3.2.jar to ./Docker-plugin-2.3.2.jar
Getting https://plugins.jetbrains.com/files/7896/25621/idea-multimarkdown.1.5.0.zip#/idea-multimarkdown-1.5.0.zip to ./idea-multimarkdown-1.5.0.zip
Building SRPM...
Building RPMs using mock...
 
SRPM and RPMs are written in /home/vitaly/projects/pycharm-community/results
```

**Comment by xvitaly** [comment]
Added Fedora's spectool into repository to build on TravisCI under Ubuntu. Now downloading sources works fine build build failes with error:
```
Building RPMs using mock...
ERROR: Could not find required config file: /etc/mock/default.cfg
ERROR:   Did you forget to specify the chroot to use with '-r'?
```

**Pull request merged**